src/test/ui/sanitize/hwaddress.rs

```diff
@@ -0,0 +1,19 @@
+// needs-sanitizer-support
+// needs-sanitizer-hwaddress
```

Comment:
The implementation of `needs-sanitizer-hwaddress` would go somewhere around here: https://github.com/rust-lang/rust/blob/099f27b6cbf10e31d41e4a2e6cb55ec39dbb0527/src/tools/compiletest/src/header.rs#L88-L91

Reply:
Done. Thanks for the pointer!